src/components/ListChampions.jsx

```diff
@@ -1,33 +1,33 @@
 import { useState, useEffect } from 'react';
+import CardChampions from './CardChampions';
 
 function List() {
   const [champions, setChampions] = useState([]);
 
   useEffect(() => {
     // Récupération de la liste de tous les champions depuis l'API Data Dragon
-    fetch('http://ddragon.leagueoflegends.com/cdn/13.6.1/data/en_US/champion.json')
+    fetch('https://ddragon.leagueoflegends.com/cdn/13.6.1/data/en_US/champion.json')
```

Comment:
```suggestion
    fetch('//ddragon.leagueoflegends.com/cdn/13.6.1/data/en_US/champion.json')
```
Permet au navigateur de choisir automatiquement `https` ou `http`

Reply:
👍🏼